### DISCUSSION TITLE
RichEditor/Trix side by side images

### DISCUSSION THREAD

**Discussion by tiagof** [filamentphp/filament#3995] [Help]
Hi.

Currently I cannot insert images side by side - like we see here: https://trix-editor.org/
Found that the `trix-content` class is missing from the `<trix-editor>` element and, when added, it works.

Is there a reason for this class not being included?

**Comment by themahdavi** [comment]
Hello 
If your are added on your own project and work fine please send PR.

**Reply by tiagof** [reply]
Hi @themahdavi .

It works fine but:
- the project is still under development so no production feedback yet.
- adding this class can "break" things for some users: today the images are displayed vertically; with this class/PR the images could eventually be displayed side-by-side, depending on the actual trix-editor content, which may not be desired.

Still, i'll raise the PR!

**Comment by tiagof** [comment]
I guess I'll hold the PR for now. There are other strange things I guess tailwind is to "blame" for.

I'm seeing images left aligned, when they should be center aligned.

With this in filament/backoffice:

<img width="1070" alt="Screenshot 2022-09-14 at 14 22 43" src="https://user-images.githubusercontent.com/1729910/190165913-de926cca-1a8b-4c81-a08d-98e131339dac.png">



I get this in the frontend (WITHOUT tailwind):

<img width="1210" alt="Screenshot 2022-09-14 at 14 24 53" src="https://user-images.githubusercontent.com/1729910/190166286-974578c7-ff05-4a2a-ab26-a15114f6ed7a.png">



And this in the frontend (WITH tailwind):

<img width="1222" alt="Screenshot 2022-09-14 at 14 25 52" src="https://user-images.githubusercontent.com/1729910/190166468-8f401658-cdd8-4a31-b1de-a74cb39ed7da.png">


Does anyone get this behaviour as well?
I mean, even if I use tailwind in the frontend to maintain consistency, the images are left aligned when they shouldn't be.

Thanks
